Makefile

```diff
@@ -3,7 +3,6 @@
 
 PROJECT_NAME             := upf-epc
 VERSION                  ?= $(shell cat ./VERSION)
-GO_FILES                 := $(shell find . -type d \( -path ./pfcpiface/vendor -o -path ./pfcpiface/bess_pb  \) -prune -o -name '*.go' -print)
```

Comment:
Vendoring is gone and `go fmt ./...` does the job of finding .go files on its own.

Reply:
@osinstom I see you added this line, let me know if I missed something here

Reply:
This will make `go fmt` format *.pb.go files too, but since they seem to be properly formatted, I'm fine with using `go fmt ./...`